care/facility/api/viewsets/file_upload.py

```diff
@@ -18,7 +24,7 @@ class FileUploadFilter(filters.FilterSet):
 
 
 class FileUploadViewSet(
-    CreateModelMixin, RetrieveModelMixin, ListModelMixin, GenericViewSet,
+    CreateModelMixin, RetrieveModelMixin, ListModelMixin, UpdateModelMixin, DestroyModelMixin, GenericViewSet,
```

Comment:
destroy should destroy the file from AWS

Reply:
In the issue it was written that we needed soft deletion for the files, in that case we shouldn't delete the file from AWS right?

Reply:
soft delete in the database might be what he meant, the file needs to be deleted still ( Because it makes 0 sense to keep a file with no reference and containing actual confidential data ).
cc @gigincg

Reply:
@iamsdas had a conversation with the team, lets do the soft delete itself, they want it for audit reasons.